.travis.yml

```diff
@@ -32,7 +32,7 @@ install:
     # Copy in the sudoers file
     - sudo cp /tmp/sudoers.tmp /etc/sudoers 
     # Now that sudo is good to go, finish installing dependencies
-    - sudo python3 -m pip install -r requirements.txt
+    - sudo python3 -m pip install -r requirements_linux.txt
```

Comment:
can we work on a windows test in travis?

Reply:
Thoughts? Can't run travis on windows

Reply:
I have a branch to add testing with Circle CI - I’ll message you